editor/src/core/shared/object-utils.ts

```diff
@@ -124,7 +124,7 @@ export function objectMap<T extends Record<string, unknown>, K extends keyof T,
   return mappedObj
 }
 
-export function objectMapDropNulls<T, K extends keyof T, U>(
+export function objectMapDropNulls<T extends MapLike<any>, K extends keyof T, U>(
```

Comment:
A general problem was that we can not just assume that a general T type parameter is an object.